server/routes/api/accommodations/cityAccommRoute.js

```diff
@@ -0,0 +1,19 @@
+import express from 'express';
+import accomController from '../../../controllers/accomodationController';
+import accommodationValidation from '../../../middlewares/validators/accommodation';
+import { authorizationCheck } from '../../../middlewares/authorization';
+import allow from '../../../middlewares/roleAuthorisation';
+
+const router = express.Router();
+
+router.get(
+    '/city/accommodations/:city',
```

Comment:
The same here.

Reply:
Kindly have all routes related to accommodation in one file.